### PR TITLE
Parses URLs from comments and creates pages with links.

### PR DESCRIPTION
**Pull request by vasilionjea**
Parses URLs from comments on [Issue 1](https://github.com/tastejs/TasteMusic/issues/1/) and creates pages with links. The paginated URLs can be retrieved like so: `/shared-links/?page=2`, `/shared-links/?page=3`

Also removed jQuery. Assumes only modern Browser support.

### CODE REVIEW

File: shared-links/.gitignore [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile ~/.gitignore_global
```
Comment (sindresorhus) [inline shared-links/.gitignore]: read the above

this file should be in your global local gitignore

Comment (vasilionjea) [inline shared-links/.gitignore]: Thanks! 
Changed to ignore only node_modules.

**Comment by sindresorhus** [comment]
Can you bring in Bootstrap just to get some nicer styling?

Just use the CDN:

```
//netdna.bootstrapcdn.com/bootstrap/3.1.0/css/bootstrap.min.css
```

**Comment by vasilionjea** [comment]
I've added a few basic styles - feel free to play around!

I think some things people could contribute might include:
- Design
- Adding **next** & **prev** links that navigate with the query string like: `?page=2`, `?page=3`, etc. - perhaps using `pushState`.
- Local Storage - once parsed, save the URLs locally.
- A prototype integrating with the [SoundCloud API](http://developer.soundcloud.com/docs/api/sdks#javascript).
- A prototype integrating with the [YouTube API](https://developers.google.com/youtube/getting_started)
- Possibly integrations with other APIs - (fallback can always be a simple link).
- ...

**Comment by Glavin001** [comment]
Great work, @vasilionjea! I'm hoping to be able to help out with development in the next couple weeks. Integrating with SoundCloud and YouTube and other such APIs will be very cool in the end. Looking forward to using this!

**Comment by vasilionjea** [comment]
@Glavin001 Thanks! I'm glad you're thinking of helping. 

If @sindresorhus merges the pull request perhaps others might want to help too.

**Comment by addyosmani** [comment]
Merged. Let's build on this great start :)

**Pull request merged**